src/modules/home/components/singleProperty/singlePropertyAmenity.tsx

```diff
@@ -0,0 +1,178 @@
+              }}
+            >
+              <Box>
+                <Typography variant="subtitle1" sx={{ fontWeight: 'bold' }}>
+                  {amenity.name}
+                </Typography>
+                <Typography variant="body2" color="text.secondary">
+                  {amenity.description}
+                </Typography>
+              </Box>
+            </Box>
+          ))}
+        </Box>
+
+        {amenities.length > 10 && (
+          <Button onClick={handleShowAll} sx={{ mt: 2 }} variant="outlined" color="primary">
+            {showAll ? 'Show Less' : 'Show All'}
+          </Button>
+        )}
+
+        <Divider sx={{ my: 3 }} />
+
+        <Typography variant="body1">{description}</Typography>
+
+        <Divider sx={{ my: 3, display: { md: 'none', xs: 'block' } }} />
+      </Box>
+
+      <Box
+        sx={{
+          p: 2,
+          maxWidth: 400,
+          flex: '1 1 35%',
+          border: '1px solid rgba(0, 0, 0, 0.1)',
+          mt: { xs: 0, md: 2 },
+          borderRadius: '15px',
+          boxShadow: '0px 4px 10px rgba(0, 0, 0, 0.1), 0px 2px 5px rgba(0, 0, 0, 0.06)',
+          mx: { xs: 2, md: 0 },
+        }}
+      >
+        <Typography variant="h6">${pricePerNight}</Typography>
+        <LocalizationProvider dateAdapter={AdapterDayjs}>
+          <Box sx={{ display: 'flex', flexDirection: 'row', gap: 2, mt: 2 }}>
+            <Box sx={{ flex: '1' }}>
+              <DatePicker label="Check-in" value={checkInDate} onChange={(newDate) => setCheckInDate(newDate)} />
+            </Box>
+            <Box sx={{ flex: '1' }}>
+              <DatePicker label="Check-out" value={checkOutDate} onChange={(newDate) => setCheckOutDate(newDate)} />
+            </Box>
+          </Box>
+        </LocalizationProvider>
+        <Button sx={{ mt: 2, width: '100%', background: 'rgb(219,10,98)', py: 1 }} variant="contained">
+          Reserve
+        </Button>
+        <Box sx={{ mt: 2 }}>
+          <Typography variant="body1" sx={{ display: 'flex', justifyContent: 'space-between' }}>
+            <Typography variant="body1">
+              ${pricePerNight} x {isDatePicked ? calculatePrice.diffInDays : 4} nights{' '}
+            </Typography>{' '}
+            <Typography variant="body1">${calculatePrice.totalPrice}</Typography>
+          </Typography>
+          <Typography variant="body1" sx={{ display: 'flex', justifyContent: 'space-between', mt: 1 }}>
+            <Typography variant="body1">StayHub Service Fee</Typography> <Typography variant="body1">$20</Typography>
+          </Typography>
+          <Divider sx={{ my: 2 }} />
+          <Typography variant="body1" sx={{ mt: 2, display: 'flex', justifyContent: 'space-between' }}>
+            <Typography variant="body1" sx={{ fontWeight: 'bold' }}>
+              Total
+            </Typography>{' '}
+            <Typography variant="body1" sx={{ fontWeight: 'bold' }}>
+              ${calculatePrice.totalPrice + calculatePrice.serviceFee}
```

Comment:
It is better to have separate variables like basePrice (does not include service) and totalPrice (which will include service) and avoid doing calculations inside JSX.